include/vapor/Grid.h

```diff
@@ -94,19 +94,14 @@ class VDF_API Grid {
     //! the constructor. If the parameter has less than 3 values, then
     //! number 1 will be filled.
     //!
-    DimsType GetDimensions() const
-    {
-        auto tmp = DimsType{1, 1, 1};
-        std::copy(_dims.begin(), _dims.end(), tmp.begin());
-        return tmp;
-    }
+    const DimsType &GetDimensions() const { return _dims; }
```

Comment:
The first `const` only confuses things so I suggest remove it. 
```suggestion
    DimsType &GetDimensions() const { return _dims; }
```

Reply:
Woop, sorry missed the & sign... All good.